hibernate-core/src/main/java/org/hibernate/query/SelectionQuery.java

```diff
@@ -493,6 +514,30 @@ default Stream<R> stream() {
 	@Incubating
 	SelectionQuery<R> descending(SingularAttribute<? super R, ?> attribute);
```

Comment:
Shouldn't `ascending`/`descending` be removed in favor of `sort`? It feels like they no longer bring much...

Reply:
@yrodiere You win. They are now gone.